Sealnote/build.gradle

```diff
@@ -71,15 +71,18 @@ tasks.withType(JavaCompile) {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile fileTree(dir: "$buildDir/native-libs", include: 'native-libs.jar')
+    //compile fileTree(dir: "$buildDir/native-libs", include: 'native-libs.jar')
 
-    compile 'com.android.support:support-v4:20.0.+'
+    compile 'com.android.support:support-v4:25.1.1'
     compile 'com.etsy.android.grid:library:1.0.4'
     compile 'com.github.gabrielemariotti.cards:library:1.6.0'
     compile 'com.github.gabrielemariotti.cards:library-extra:1.6.0'
     compile 'com.nhaarman.listviewanimations:library:2.6.0'
 
-    compile 'com.github.codechimp-org.apprater:library:1.0.+'
+    compile 'com.github.codechimp-org.apprater:library:1.0.32'
+    compile 'net.zetetic:android-database-sqlcipher:3.3.1-2@aar'
```

Comment:
I added this dependency instead of native libraries and sqlcipher.jar.
SQLCipher for Android is distributed via the AAR package format now. Adding this dependency is enough.